Packages/com.unity.multiplayer.samples.coop/Utilities/Net/SpawnObjectData.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.Utilities
+{
+    public class SpawnObjectData : MonoBehaviour
```

Comment:
this could still be merged right? or am I reviewing too early? :P

Reply:
I've got no more modifications I wish to make, so yeah 😄

Reply:
So in this case, yeah, that data could be merged with the spawner (and no need for list) --> it'd make this even simpler